WORKSPACE

```diff
@@ -831,7 +831,7 @@ new_git_repository(
     name = "model_api",
     remote = "https:///github.com/openvinotoolkit/model_api/",
     build_file = "@_model-api//:BUILD",
-    commit = "eb9fcfb1e1eebc047ff144707f76203b132e1aa6" # master Jun 24 15:02:17 2024 [cpp] Fix num classes check
+    commit = "449ced0ac2841dd61f54e3346dea4fcfe9f34703" # v0.2.5 Sep 27, 2024
```

Comment:
We need model api to be at least:
25c88f8fd1ebe08447aca9a959a7a5f37751867e

Reply:
resolved by Rafal